### PR TITLE
Nala base reporter CircleCi url update

### PR DESCRIPTION
**Pull request by amauch-adobe**
updated circleci link for nala test results report printout.

### CODE REVIEW

**Review by Dli3 — APPROVED** [review]
Looks like there's a few linter errors.

File: utils/reporters/base-reporter.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { sendSlackMessage } from '../../libs/slack.js';
```
[the remark this answers is not in the document]
Comment (JackySun9) [reply utils/reporters/base-reporter.js:1]: what's this?

**Review by JackySun9 — CHANGES_REQUESTED** [review]
Can you fix eslint errors?

**Review by JackySun9 — APPROVED** [review]

**Comment by amauch-adobe** [comment]
@JackySun9 , the slack library is used to send the JSON payload to slack webhook for nala test results (#nala-test-results slack channel).

Where I had only made the change to the file to update the circleci url text for the base reporter, I wasn't planning on addressing all of the Lint errors.  I think it would be good for us to create a ticket in the Jira Epic to audit all of the files using the Linter so that we can fix previous files that are already in the repository.  @skumar09 what do you think.

**Pull request merged**